packages/expect-utils/src/types.ts

```diff
@@ -6,4 +6,8 @@
  *
  */
 
-export type Tester = (a: any, b: any) => boolean | undefined;
+export type Tester = (
+  a: any,
+  b: any,
+  customTesters: Array<Tester>,
```

Comment:
is it a breaking change for consumers of Tester that this always takes an array? should it be optional?

Reply:
I believe what this is saying is that a function that implements Tester will be invoked with 3 arguments, but it doesn't have to use all of them. So this is still valid, `f` just ignores the third argument.
```ts
const f: Tester = (a, b) => { /* ... */ }
```

The [sample tester used in tests does this](https://github.com/facebook/jest/pull/13654/files#diff-883a0dc67ba4e2fbd2a2d6495cfaaaceb2e4d53b53b13fcdaa7fad82fee88742R44-R47) and TypeScript seems happy with it. If you'd feel making it optional will be safer, I'm happy to make the change. Will just lead to some unnecessary null checks in testers (I don't think we'd ever invoke it without the third argument), but perhaps that is okay.

Reply:
Fair enough! 🙂